sdk/src/main/java/io/customer/sdk/data/request/Device.kt

```diff
@@ -9,7 +9,7 @@ import java.util.*
 data class Device(
     @field:Json(name = "id") val token: String,
     val platform: String = "android",
-    val lastUsed: Date,
+    @field:Json(name = "last_used") val lastUsed: Date?,
```

Comment:
Can you please add a comment to explain why we made it nullable for future reference?